### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#489] [Esami & Esoneri M-Z]
### Traccia

<img width="623" alt="image" src="https://github.com/user-attachments/assets/7b3488a6-be6f-467d-baf2-4630c25a1963">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/25e2db13-01ad-4bab-ac0b-0f3fca609621)
